ext/readline/readline.c

```diff
@@ -305,7 +305,7 @@ PHP_FUNCTION(readline_info)
 			oldval = rl_erase_empty_line;
 			if (value) {
 				convert_to_long_ex(value);
-				rl_erase_empty_line = Z_LVAL_PP(value);
+				rl_erase_empty_line = Z_LVAL_P(*value);
```

Comment:
This code is being a `HAVE_ERASE_EMPTY_LINE` guard, which `config.m4` never sets. So currently this is just dead code. We should either completely remove it or add the necessary checks in the build system.